client/components/CellGrid/cellGrid.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import cssModules from 'react-css-modules';
+import PropTypes from 'prop-types';
+import { getMinutes } from '../../util/time-format';
```

Comment:
Following the style we have been using, this import and the next one should be grouped together, separated from the other imports by a blank line.